real_estate/models/estate_property.py

```diff
@@ -0,0 +1,143 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+import datetime
+
+class EstateProperty(models.Model):
+
+    _name = 'estate.property',
+    _description = 'Real Estate Property'
```

Comment:
```suggestion
    _description = "Real Estate Property"
```
`''` should be used for technical strings and `""` should be used for user readable strings